test/code_size/embind_hello_wasm.json

```diff
@@ -1,10 +1,10 @@
 {
   "a.html": 552,
   "a.html.gz": 380,
-  "a.js": 9593,
-  "a.js.gz": 4230,
-  "a.wasm": 7615,
-  "a.wasm.gz": 3471,
-  "total": 17760,
-  "total_gz": 8081
+  "a.js": 9879,
+  "a.js.gz": 4288,
+  "a.wasm": 7580,
+  "a.wasm.gz": 3449,
+  "total": 18011,
+  "total_gz": 8117
```

Comment:
Sad that these numbers go up and not down :(

Reply:
Embind didn't support 64bit types before, so now there's some more JS code to handle bigint types. I'm guessing a real world example would show a very small change or improvement.

Reply:
Yeah, most of the other changes are going down. And we'll have more improvements to come with the other features.